ncdns 0.1.2: build: ncdomain/convert_tls.go:125:28: undefined: "github.com/namecoin/x509-compressed/x509".ParsePKIXPublicKey

Hi, I'm trying to update the Arch Linux package to 0.1.2 but Option B doesn't work for me.

First, `go mod init` fails with:
```
$ export GOPATH=$PWD/src/_build
$ cd src/x509-compressed
$ go mod init
go: cannot determine module path for source directory /PATHTO/ncdns/src/x509-compressed (outside GOPATH, module path must be specified)
```
I tried the following, which makes this part of the build suceed, but fails later:
```
go mod init github.com/namecoin/x509-compressed
```
later, when building ncdns:
```
ncdomain/convert_tls.go:125:28: undefined: "github.com/namecoin/x509-compressed/x509".ParsePKIXPublicKey
```

What used to work in an older version:
```
   go get -t  -d github.com/namecoin/ncdns/...

  pushd $GOPATH/src/github.com/namecoin/x509-signature-splice
  go generate github.com/namecoin/x509-signature-splice/...
  popd

  go get -t github.com/namecoin/ncdns/...
```

What should I be setting GOPATH to?

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -29,14 +29,14 @@ task:
         - cd ../
         - git clone https://github.com/namecoin/x509-compressed.git
         - cd x509-compressed
-        - go mod init
+        - go mod init github.com/namecoin/x509-compressed
         - go mod tidy
         - go generate ./...
         - go mod tidy
         - go install -v ./...
       fetch_script:
         - cd $(go env GOPATH)/src/github.com/"$CIRRUS_REPO_FULL_NAME"
-        - go mod init
+        - go mod init github.com/"$CIRRUS_REPO_FULL_NAME"
         - go mod edit -replace github.com/coreos/go-systemd=github.com/coreos/go-systemd/v22@latest -replace github.com/namecoin/x509-compressed=../x509-compressed
         - go mod tidy
   lint_script:
@@ -109,14 +109,14 @@ task:
         - cd ../
         - git clone https://github.com/namecoin/x509-compressed.git
         - cd x509-compressed
-        - go mod init
+        - go mod init github.com/namecoin/x509-compressed
         - go mod tidy
         - go generate ./...
         - go mod tidy
         - go install -v ./...
       fetch_script:
         - cd $(go env GOPATH)/src/github.com/"$CIRRUS_REPO_FULL_NAME"
-        - go mod init
+        - go mod init github.com/"$CIRRUS_REPO_FULL_NAME"
         - go mod edit -replace github.com/coreos/go-systemd=github.com/coreos/go-systemd/v22@latest -replace github.com/namecoin/x509-compressed=../x509-compressed
         - go mod tidy
         # Get the test suite
@@ -165,14 +165,14 @@ task:
         - cd ../
         - git clone https://github.com/namecoin/x509-compressed.git
         - cd x509-compressed
-        - go mod init
+        - go mod init github.com/namecoin/x509-compressed
         - go mod tidy
         - go generate ./...
         - go mod tidy
         - go install -v ./...
       fetch_script:
         - cd $(go env GOPATH)/src/github.com/"$CIRRUS_REPO_FULL_NAME"
-        - go mod init
+        - go mod init github.com/"$CIRRUS_REPO_FULL_NAME"
         - go mod edit -replace github.com/coreos/go-systemd=github.com/coreos/go-systemd/v22@latest -replace github.com/namecoin/x509-compressed=../x509-compressed
         - go mod tidy
   build_script:

--- a/README.md
+++ b/README.md
@@ -112,7 +112,7 @@ Option B: Using Go build commands with Go modules (works on any platform with Ba
 2. Run the following in the ncdns directory to set up Go modules:
    
    ~~~
-   go mod init
+   go mod init github.com/namecoin/ncdns
    go mod edit -replace github.com/coreos/go-systemd=github.com/coreos/go-systemd/v22@latest -replace github.com/namecoin/x509-compressed=../x509-compressed
    go mod tidy
    ~~~

